hugo/content/blog/2021-03-20-weblate-translation/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Switch to Weblate as Translation Platform
+author: Jan Klass
+date: 2021-05-10
+categories:
+  - News
+---
+The Mumble Client is available in many languages. For the translation process we recently moved from the Transifex online platform to Weblate. In this post we describe our translation history, reasoning, and current approach.
+
+Translations now happen in the [Mumble Client project on Hosted Weblate](https://hosted.weblate.org/projects/mumble/mumble-client/).
+<!--more-->
+
+## History - From local to online
+
+Our Mumble client is making use of the Qt library which provides various features, and amongst them a system for localizing.
+
+In the source folder `src/mumble` we produce a `.ts` file for each language, which contains the source texts and corresponding translated text. These files could be edited manually with a text or XML editor, or a translation application.
+
+To lower the barrier for potential translators we moved to [Transifex](https://www.transifex.com/), a web based translation platform in 2014. Using a web platform contributors and anyone else can access translation information with a web browser, which made it a lot easier to work with. It also adds a few collaborative features that can be used.
+
+To integrate the third party service we implemented a bot ([MumbleTransifexBot](https://github.com/mumble-voip/MumbleTransifexBot)) that would pull translation updates and create pull requests on GitHub for us. Once established this significantly reduced the work necessary for maintaining this system and integrating translation changes.
+
+## Reevaluation
+
+In the 6 years since we switched to Transifex, the translation platform changed significantly. Good alternatives appeared, and we took some discussion from a issue ticket as initiation for a reevaluation of our situation and opportunities.

```

Comment:
```suggestion
In the 6 years since we switched to Transifex, the translation platform changed significantly. Good alternatives appeared, and we took some discussion from an issue ticket as initiation for a reevaluation of our situation and opportunities.

```